build/ci/release.yml

```diff
@@ -577,16 +610,47 @@ tasks:
       - func: "generate msi"
         vars:
           TOOL_NAME: ${tool_name}
-          RELEASING: true
+      - func: "update choco"
+        vars:
+          TOOL_NAME: ${tool_name}
```

Comment:
can't really test this without releasing so need to confirm if republishing old releases is safe